documentation/continuousIntegration/README.md

```diff
@@ -0,0 +1,123 @@
+# Connect Continuous Integration with GitHub
+In this workshop, we will introduce you to 2 CI services which are...
+* CircleCI
+* TravisCI 
+## CircleCI
+check out [this document](https://circleci.com/docs/2.0/language-guides/) for different setting type for your favorite project languages
+
+For us, we will use Java with gradle.
+
+* Create .circleci in your project root directory (if you have been following the tutorial without renaming anything, your root will be ci-hands-on-workshop).
```

Comment:
.circleci is a directory.
Maybe we need to clarify here a bit ?

Reply:
fixed